v3/plugins/redhat/mta-vscode-extension/0.0.58/meta.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v2
```

Comment:
This change can be removed -- `meta.yaml` files are generated via the `che-theia-plugins.yaml` file now.